app/lib/search_query_transformer.rb

```diff
@@ -69,11 +80,33 @@ def initialize(prefix, operator, phrase)
     end
   end
 
+  class PrefixClause
+    attr_reader :filter, :operator, :term
+
+    def initialize(prefix, term)
+      @operator = :filter
+      case prefix
+      when 'by'
```

Comment:
I would prefer `from` instead of `by`. It seems to be used in both Discord search and Twitter search!

Reply:
Ok. I'll change this later today